src/test/suite/extensionFunctions.test.ts

```diff
@@ -75,4 +76,24 @@ suite('Extension Functions Test Suite', () => {
 			assert.equal(returnBool, false);
 		});
 	});
+
+	test('test to make sure we can get the file path from a whole Uri', async function() {
+		const shouldEndWith = 'vscode-didact/demo/didact-demo.didact.md';
+		const shouldEndWithRaw = 'vscode-didact/master/demo/didact-demo.didact.md';
+
+		const textUri = vscode.Uri.parse('vscode://redhat.vscode-didact?extension=redhat.vscode-didact/demo/didact-demo.didact.md');
+		const rtnUri = extensionFunctions.handleVSCodeDidactUriParsingForPath(textUri);
+		console.log('---------------handleVSCodeDidactUriParsingForPath returned (1) = ' + rtnUri);
```

Comment:
I like having the log for the tests, since it gives a bit more diagnostic info. But if we split it the way you suggest in the previous comment, that does the same thing.